test/tests/neutronics/triggers/multi_rel_err.i

```diff
@@ -22,15 +22,20 @@
   power = 100.0
   batches = 20
   cell_level = 0
-  tally_type = cell
-  tally_blocks = '100'
   normalize_by_global_tally = false
 
-  tally_score = 'damage_energy kappa_fission'
-
-  tally_trigger = 'rel_err rel_err'
-  tally_trigger_threshold = '2e-2 2e-2'
   max_batches = 200
+
+  [Tallies]
+    [Cell]
+      type = CellTally
+      tally_blocks = '100'
```

Comment:
What do you think of shortening some of the names of these parameters, now that they're nested inside these blocks? Users may find the following more straightforward/faster, but perhaps there could be a good reason not to change the names, to minimize effort to port things over.

```
  [Cell]
      type = CellTally
      blocks = '100'
      score = 'damage_energy kappa_fission'

      trigger = 'rel_err rel_err'
      trigger_threshold = '2e-2 2e-2'
  []
```

Reply:
I like the idea quite a bit, though I was hesitant to change parameter names precisely because of that reason (and to minimize the effort required to update all of the tests and tutorials). If you think it's worthwhile, I can shorten the syntax.